src/core/Akka/IO/Udp.cs

```diff
@@ -556,23 +559,17 @@ private void OnComplete(object sender, SocketAsyncEventArgs e)
         {
             var actorRef = e.UserToken as IActorRef;
             actorRef?.Tell(ResolveMessage(e));
+            SocketEventArgsPool.Release(e);
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private Udp.SocketCompleted ResolveMessage(SocketAsyncEventArgs e)
+        private static Udp.SocketCompleted ResolveMessage(SocketAsyncEventArgs e)
         {
             switch (e.LastOperation)
             {
                 case SocketAsyncOperation.Receive:
                 case SocketAsyncOperation.ReceiveFrom:
                     return new Udp.SocketReceived(e);
-                case SocketAsyncOperation.Send:
```

Comment:
These are never used. Removed to remove red herrings and possible problem in the future.